src/core/mac/mac.cpp

```diff
@@ -1316,7 +1316,7 @@ void Mac::ReceiveDoneTask(Frame *aFrame, ThreadError aError)
         }
 
         srcaddr.mLength = sizeof(srcaddr.mExtAddress);
-        memcpy(&srcaddr.mExtAddress, &neighbor->mMacAddr, sizeof(srcaddr.mExtAddress));
+        srcaddr.mExtAddress = neighbor->GetExtAddress();
```

Comment:
:+1: Kind of cool that c++ pass-by-reference assignment just does the copy for you.